pkg/controller/handler.go

```diff
@@ -137,6 +138,42 @@ func (h *handler) CreateServiceInstance(in *servicecatalog.Instance) (*serviceca
 	return h.apiClient.Instances(in.ObjectMeta.Namespace).Update(in)
 }
 
+func (h *handler) DeleteServiceBinding(sb *servicecatalog.Binding) error {
+	dts := metav1.Now()
```

Comment:
Can we get a marker comment here saying which parts (or just the whole thing) has some TPR-specific logic in it?